lte/cloud/go/services/subscriberdb/obsidian/handlers/handlers_test.go

```diff
@@ -474,6 +474,241 @@ func TestListSubscribers(t *testing.T) {
+	test_utils.ReportState(t, ctx, lte.MMEStateType, "IMSI1234567890", &mmeState, serdes.State)
+	spgwState := state.ArbitraryJSON{"spgw": "foo"}
+	test_utils.ReportState(t, ctx, lte.SPGWStateType, "IMSI1234567890", &spgwState, serdes.State)
+	s1apState := state.ArbitraryJSON{"s1ap": "foo"}
+	test_utils.ReportState(t, ctx, lte.S1APStateType, "IMSI1234567890", &s1apState, serdes.State)
+	// Report 2 allocated IP addresses for the subscriber
+	mobilitydState1 := state.ArbitraryJSON{
+		"ip": map[string]interface{}{
+			"address": "wKiArg==",
+		},
+	}
+	mobilitydState2 := state.ArbitraryJSON{
+		"ip": map[string]interface{}{
+			"address": "wKiAhg==",
+		},
+	}
+	test_utils.ReportState(t, ctx, lte.MobilitydStateType, "IMSI1234567890.oai.ipv4", &mobilitydState1, serdes.State)
+	test_utils.ReportState(t, ctx, lte.MobilitydStateType, "IMSI1234567890.magma.apn", &mobilitydState2, serdes.State)
+	directoryState := directorydTypes.DirectoryRecord{LocationHistory: []string{"foo", "bar"}}
+	test_utils.ReportState(t, ctx, orc8r.DirectoryRecordType, "IMSI1234567890", &directoryState, serdes.State)
+
+	expectedResult.Subscribers = map[string]*subscriberModels.Subscriber{
+		"IMSI0987654321": {
+			ID: "IMSI0987654321",
+			Lte: &subscriberModels.LteSubscription{
+				AuthAlgo:   "MILENAGE",
+				AuthKey:    []byte("\x22\x22\x22\x22\x22\x22\x22\x22\x22\x22\x22\x22\x22\x22\x22\x22"),
+				AuthOpc:    []byte("\x22\x22\x22\x22\x22\x22\x22\x22\x22\x22\x22\x22\x22\x22\x22\x22"),
+				State:      "ACTIVE",
+				SubProfile: "foo",
+			},
+			Config: &subscriberModels.SubscriberConfig{
+				Lte: &subscriberModels.LteSubscription{
+					AuthAlgo:   "MILENAGE",
+					AuthKey:    []byte("\x22\x22\x22\x22\x22\x22\x22\x22\x22\x22\x22\x22\x22\x22\x22\x22"),
+					AuthOpc:    []byte("\x22\x22\x22\x22\x22\x22\x22\x22\x22\x22\x22\x22\x22\x22\x22\x22"),
+					State:      "ACTIVE",
+					SubProfile: "foo",
+				},
+			},
+			ActiveApns: subscriberModels.ApnList{apn1},
+		},
+		"IMSI0987654322": {
+			ID: "IMSI0987654322",
+			Lte: &subscriberModels.LteSubscription{
+				AuthAlgo:   "MILENAGE",
+				AuthKey:    []byte("\x22\x22\x22\x22\x22\x22\x22\x22\x22\x22\x22\x22\x22\x22\x22\x22"),
+				AuthOpc:    []byte("\x22\x22\x22\x22\x22\x22\x22\x22\x22\x22\x22\x22\x22\x22\x22\x22"),
+				State:      "ACTIVE",
+				SubProfile: "foo",
+			},
+			Config: &subscriberModels.SubscriberConfig{
+				Lte: &subscriberModels.LteSubscription{
+					AuthAlgo:   "MILENAGE",
+					AuthKey:    []byte("\x22\x22\x22\x22\x22\x22\x22\x22\x22\x22\x22\x22\x22\x22\x22\x22"),
+					AuthOpc:    []byte("\x22\x22\x22\x22\x22\x22\x22\x22\x22\x22\x22\x22\x22\x22\x22\x22"),
+					State:      "ACTIVE",
+					SubProfile: "foo",
+				},
+			},
+			ActiveApns: subscriberModels.ApnList{apn2},
+		},
+	}
+	expectedResult.NextPageToken = "Cg5JTVNJMDk4NzY1NDMyMg=="
+
+	// Test paginated requests
+	tc = tests.Test{
+		Method:         "GET",
+		URL:            testURLRoot + "?page_size=2&page_token=",
```

Comment:
Can you also test
- when `page_token` parameter isn't specified at all
- making the same request twice, with same page token, and should have same output